drafts/str4d-overwinter-activation/draft1.rst

```diff
@@ -0,0 +1,232 @@
+::
+
+  ZIP: ???
+  Title: Network Upgrade Mechanism
+  Author: Jack Grigg <jack@z.cash>
+  Comments-Summary: No comments yet.
+  Category: Process
+  Created: 2018-01-08
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "MUST NOT", "SHOULD", "SHOULD NOT", and "MAY" in this document are to be interpreted as
+described in RFC 2119.
+
+The following terms are interpreted as follows:
+
+Branch
```

Comment:
This is a little fuzzy.  For it to be well-defined it seems the branch id needs to be included in the transaction. And then you could say a branch is a sequence of blocks all with the same branch_id.  But in the ZIP you don't talk about the branch_id being included in the transactions.  
And what you say about wipe out protection (comitting to branch id in block header) seems to suggest the branch_id is not included in the transaction for now (in Overwinter) .   
Cause if it was included in every transaction, that would be the wipe out protection

Reply:
The *consensus* branch ID *is* included in the transaction (in every signature).

The comment I make later is referring to the fact that if a block does not contain any signature (which is the case for coinbase transactions), it would not currently bind to a consensus branch ID and thus could be susceptible to a wipe-out. Overwinter avoids this because it is changing the transaction format, but it *could* be the case in a future network upgrade that doesn't change the transaction format. This is just pointing out that future upgrades may need to do more (like bind to a consensus branch ID in the block header).

Reply:
Remind me - the consensus branch id included in the signature is the same, or not necessarily the same, as the branch_id field?
Anyways this should be made more explicit, e.g. say that "following common consensus rules" means "signatures valid with the same branch_id", and perhaps reference to signature hash algorithm